assets/js/buttons-examples.js

```diff
@@ -154,6 +154,20 @@ $(document).ready(function () {
       label: "Animated Buttons",
       classes: "",
       children: [
+        {
+          type: "gradient-border",
+          label: "Animated Gradient Border",
```

Comment:
remove the animated from the label and place it then at the correct alphabetical order.